lib/experimental/Navigation/Tabs/index.tsx

```diff
@@ -30,19 +30,27 @@ export const BaseTabs: React.FC<TabsProps> = ({ tabs, secondary = false }) => {
       asChild
       aria-label={secondary ? "primary-navigation" : "secondary-navigation"}
     >
-      {tabs.map(({ label, ...props }, index) => (
-        <TabNavigationLink
-          key={index}
-          active={activeTab?.href === props.href}
-          href={props.href}
-          secondary={secondary}
-          asChild
-        >
-          <Link role="link" {...props}>
-            {label}
-          </Link>
+      {tabs.length === 1 ? (
+        <TabNavigationLink disabled secondary={secondary} className="p-0">
```

Comment:
Accessibility wise it is not great. It is disabled, hence will be pronounced as a disabled element. However I'm not sure it can be selected at all. 

It is also not copyable, it is counterintuitive when you deal with just a text

Reply:
Ah, you're right, it's not semantically correct either, it's not really disabled. Changed it to a li element, so it allows text selection/copying.